plugins/package-managers/go/src/funTest/kotlin/utils/GoSupportFunTest.kt

```diff
@@ -0,0 +1,45 @@
+/*
```

Comment:
Commit message:

> are now properly escaped

Would you have a reference to some Go documentation web page that defines the escaping rules?

Reply:
Done. Added link to go sources where the rules for escaping are described.